src/immutable/state/index.ts

```diff
@@ -16,11 +16,11 @@ export * from './dashboard';
 export * from './users';
 
 export interface RootStateProps {
-  admin: AdminStateRecord;
-  core: CoreStateRecord;
-  dashboard: DashboardStateRecord;
-  router: RouterState;
-  users: UsersStateRecord;
+  admin?: AdminStateRecord;
+  core?: CoreStateRecord;
+  dashboard?: DashboardStateRecord;
+  router?: RouterState;
+  users?: UsersStateRecord;
```

Comment:
I'm surprised that these need to be optional. Why is that?

Reply:
Because they didn't have a default value. Now they do!